adobe/react-spectrum
# fix(#2476): Picker: HTML5 autoComplete prop does not get added to HiddenSelect within Picker

Closes #2476

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 2476](https://github.com/adobe/react-spectrum/issue/2476).

## 📝 Test Instructions:

1. Open https://reactspectrum.blob.core.windows.net/reactspectrum/087c0f57683c0681f2d4f66cefedf3c8f052ddc0/storybook/index.html?path=/story/form--fields-with-autocomplete-property
2. Open Web Inspector and verify that for each `input` within `TextField`, `TextArea` and the hidden `select` element within `Picker` an `autocomplete` attribute exists. For example, the hidden `select` element within the State `Picker` should have `autocomplete="billing address-level1"` within the Billing Address fieldset and `autocomplete="shipping address-level1"` within the  Shipping Address fieldset.

## 🧢 Your Project:

Adobe/Accessibility
